tests/nested-root-workspace/target/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
I did this because it's simpler to run on `tests/`, the alternative here would probably be to copy all projects to a `tempdir`, create the `target` folder for each one of them, and run `sweep -r -i` in this tempdir instead of `tests/`.